web/modules/custom/rsvplist/src/EnablerService.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\rsvplist;
+
+use Drupal;
+use Drupal\Core\database\Connection;
+use Drupal\node\Entity\Node;
+use Exception;
+
+/**
+ * EnablerService Class.
+ */
+class EnablerService {
+
+  /**
+   * @var Connection $database_connection
+   *   A database connection object.
+   */
+  protected Connection $database_connection;
+
+  /**
+   * @param Connection $connection
+   *   Takes the Connection object.
+   */
+  public function __construct(Connection $connection) {
+    $this->database_connection = $connection;
+  }
+
+  /**
+   * Checks if the Current node is RSVP Enabled or not.
+   *
+   * @param \Drupal\node\Entity\Node $node
+   *   Takes the Node Object.
+   * 
+   * @return bool
+   *   Returns True or False Based on the RSVP Enabled status.
+   */
+  public function isEnabled(Node $node) {
+
+    if ($node->isNew()) {
+      return FALSE;
+    }
+
+    try {
+      $rsvp_enabled = $this->database_connection->select('rsvplist_enabled', 're');
+      $rsvp_enabled->fields('re', ['nid']);
+      $rsvp_enabled->condition('nid', $node->id());
+      $result = $rsvp_enabled->execute();
+
+      return !(empty($result->fetchCol()));
+
+    }
+    catch (Exception $e) {
+      Drupal::messenger()->addMessage(t("Sorry Drupal Couldn't Connect to database"));
+      return NULL;
```

Comment:
As return type mentioned in docblock is null, I think it should be true or false. In case using both change docblock to bool|null return type.

Reply:
This is fine. No need for change.